repxe_host.rb

```diff
@@ -496,7 +496,11 @@ def graceful_shutdown(chef_env, vm_entry)
   unless options[:newmachine]
```

Comment:
We should probably also remove the `newmachine` option (`-n`) with this change.

Reply:
Makes sense at this point it is useless